docs/vm/cpu-pinning.md

```diff
@@ -0,0 +1,112 @@
+
+Harvester supports VM CPU pinning. To use this feature, you must first enable the CPU Manager on the nodes, and then enable CPU pinning when you create the VM.
+
+## Kubernetes CPU Manager
+
+The [CPU Manager](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/) feature improves CPU resource allocation in Kubernetes clusters, ensuring that workloads with strict performance needs receive stable and predictable CPU resources. This is especially important for high-performance and latency-sensitive applications.
+
+Harvester uses the `static` CPU Manager policy when the CPU Manager is enabled. This policy manages a shared pool of CPUs that initially includes all CPUs on nodes with the following configuration:
+
+- Pods in the [`Guaranteed`](https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#guaranteed) quality of service (QoS) class that request whole CPU cores (for example, CPU: "2") are assigned dedicated CPUs. These CPUs are "pinned" to the pod and are removed from the shared CPU pool.
+- Pods in the [`Burstable`](https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#burstable) and [`BestEffort`](https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#besteffort) QoS classes share the remaining CPUs in the shared pool.
+
+### Calculation of Shared CPU Pool
+
+Harvester reserves CPU resources for system-level operations based on the [GKE formula](https://cloud.google.com/kubernetes-engine/docs/concepts/plan-node-sizes#cpu_reservations), with the [`systemReserved`](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved) and [`kubeReserved`](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#kube-reserved) values allocated in a 2:3 ratio.
+
+Example (node with 16 CPU cores):
+```
+systemReserved: 408 millicores
+kubeReserved: 612 millicores
+```
+In this example, approximately 15 cores (14980 millicores) are available for workloads.
+
+When a VM (pod) in the Guaranteed QoS class requests 4 CPUs, 4 cores are dedicated to that VM. Pods in the other QoS classes share the remaining 11 cores in the shared pool.
+
+## Enable and Disable CPU Manager
+
+When you enable the CPU Manager, Harvester sets the CPU Manager policy to `static`. When you disable the feature, Harvester switches the CPU Manager policy back to `none`. 
+
+You must enable or disable the CPU Manager on each node separately.
+
+1. On the Harvester UI, go to **Hosts**.
+
+1. Locate the node in the list, and then select **⋮** > **Enable CPU Manager** or **Disable CPU Manager**.
+
+Allow some time for Harvester to apply the corresponding CPU Manager policy.
+
+![enable-cpu-manager](/img/v1.4/cpu-pinning/enable-cpu-manager.png)
+![disable-cpu-manager](/img/v1.4/cpu-pinning/disable-cpu-manager.png)
+
+### Limitations
+
+- The CPU Manager cannot be enabled on the witness node.
+
+- You can enable or disable the CPU Manager on only one management node at a time.
```

Comment:
This sounds like when a user enables CPU manager on a management node, he/she can't enable another management node. It would be nice to re-phase it more clearly.